codespell_lib/data/dictionary.txt

```diff
@@ -15960,7 +15997,10 @@ decerning->discerning
 decerns->discerns
 deciaml->decimal
 deciamls->decimals
-decices->decides
+decice->device, decide,
```

Comment:
Perhaps `decile` too, although **C** and **L** are far apart on most keyboards? Plural too.

Reply:
`decice->decile` is rare, but exists. I think this is caused by accidentally using `c` twice in a row.